layouts/partials/cookie_consent.html

```diff
@@ -0,0 +1,5 @@
+<script type="text/javascript">
+    window.cookieconsent_options = {{ dict "message" "This website uses cookies." "dismiss" "Got it!" "learnMore" "More info" "link" (.Site.Params.cookie_consent_info_url | absURL)  "theme" "dark-top" | jsonify | safeJS }};
+</script>
+
+<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/1.0.10/cookieconsent.min.js"></script>
```

Comment:
I would vote for integrating it into the theme. Having something trustable which cannot change without notice feels better than external resources which may go away, be unreliable or change without notice.

Your are right with the version - I simply clicked through https://silktide.com/tools/cookie-consent/ without noticing the red banner about version 3.
Feel to change this - if it has to go into the theme I cannot help anyway (never used JavaScript - this is complete foreign territory for me).

Regarding the colours - that's a difficult decision - The displayed sentence should pull attention quickly. If it harmonises too much with the theme the whole thing may be overlooked.

Maybe the gray/dark blue palette would work fine:

`<link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.2/cookieconsent.min.css" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.2/cookieconsent.min.js"></script>
<script>
window.addEventListener("load", function(){
window.cookieconsent.initialise({
  "palette": {
    "popup": {
      "background": "#edeff5",
      "text": "#838391"
    },
    "button": {
      "background": "#4b81e8"
    }
  },
  "theme": "classic"
})});
</script>`